src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/GhDampingFunction.hpp

```diff
@@ -0,0 +1,54 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines base-class GhDampingFunction.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+/// Holds classes implementing GhDampingFunction (functions \f$R^n \to R\f$).
+namespace GeneralizedHarmonic::ConstraintDamping {
+template <size_t VolumeDim, typename Fr>
```

Comment:
can you put these forward declarations (only one so far) in a `/// \cond`? I don't want Doxygen to pick up the undocumented versions (sigh Doxygen...)